### PR TITLE
Updates to Bandwidth Integration HOW-TO based on my experience

### PR DESCRIPTION
**Pull request by rhayhow**
# Fixes # (issue)

## Description

_Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change, and any blockers that make your change a WIP_

# Checklist:

- [x] I have manually tested my changes on desktop and mobile
- [x] The test suite passes locally with my changes
- [x] If my change is a UI change, I have attached a screenshot to the description section of this pull request
- [x] [My change is 300 lines of code or less](https://github.com/MoveOnOrg/Spoke/blob/main/CONTRIBUTING.md#submitting-your-pull-request), or has a documented reason in the description why it’s longer
- [x] I have made any necessary changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] My PR is labeled [WIP] if it is in progress

### CODE REVIEW

**Comment by rhayhow** [comment]
I have shared some of the details of this pull request with @Arique1104 so they should have a sense of what the changes are.

File: docs/HOWTO_INTEGRATE_BANDWIDTH.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Bandwidth Integration
 
-Bandwidth.com is a telephone service API company. To use Bandwidth, set `DEFAULT_SERVICE=bandwidth`. The `sticky-sender` and `num-picker` service managers are required for the Bandwidth extension to work. `sticky-sender` must come before `numpicker-basic` in the `SERVICE_MANAGERS` environment variable.
+Bandwidth.com is a telephone service API company. To use Bandwidth, set `DEFAULT_SERVICE=bandwidth`. The `sticky-sender` and `numpicker-basic` service managers are required for the Bandwidth extension to work. `sticky-sender` must come before `numpicker-basic` in the `SERVICE_MANAGERS` environment variable. NOTE: Because service managers are complicated, there is not an entry for SERVICE_MANAGERS in the Spoke Documentation Hub. Just add it as an enviromental variable.
```
Comment (Arique1104) [inline docs/HOWTO_INTEGRATE_BANDWIDTH.md:3]: Here's the documentation for [Service Managers](https://github.com/MoveOnOrg/Spoke/blob/main/docs/HOWTO-use-service-managers.md)

We could have the Service Managers doc (linked above) can have a new section for `ngpvan` and beneath it, link the HOWTO Integrate Bandwidth.  I'd be happy to make those changes and push them to this branch. :)

**Review by Arique1104 — COMMENTED** [review]
That's my only addition, to add on some documentation to our Service Managers page.

**Comment by Arique1104** [comment]
@rhayhow, thank you for these changes!  You should see them reflected in the [Bandwidth Community Input Pull Request](https://github.com/MoveOnOrg/Spoke/pull/2207)

I'd love it if you could review those changes and give any feedback.

I had a lot of trouble getting your branch pulled to my local, so I ended up incorporating your edits and making a couple of markdown readability edits and I hope the final product is useful.

With your permission, I'd like to close this pull request.

File: docs/HOWTO_INTEGRATE_BANDWIDTH.md (hunk 1 above)
Comment (rhayhow) [reply docs/HOWTO_INTEGRATE_BANDWIDTH.md:3]: I will look at what is required to update the Service Managers doc. Please feel free to close the request :)

**Comment by rhayhow** [comment]
Please feel free to close the pull request.

**Pull request closed without merging**